src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4841indentation/InputClassWithChainedMethods.java

```diff
@@ -1,31 +1,31 @@
 package com.google.checkstyle.test.chapter4formatting.rule4841indentation;
 
 /** some javadoc. */
-public class ClassWithChainedMethods {
+public class InputClassWithChainedMethods {
 
-  public ClassWithChainedMethods(Object... params) {}
+  public InputClassWithChainedMethods(Object... params) {}
 
   /** some javadoc. */
   public static void main(String[] args) {
-    new ClassWithChainedMethods()
+    new InputClassWithChainedMethods()
         .getInstance("string_one")
     .doNothing("string_one".trim(), "string_two");
     // violation above ''method call' child has incorrect indentation level 4, expected .* 8.'
 
-    int length =
-    new ClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing").length();
+    String test =
+    new InputClassWithChainedMethods("param1", "param2").getInstance().doNothing("nothing");
```

Comment:
had to remove trailing `.length()` because line was crossing 100 width column limit, this have no impact on our testing coverage